Assets/Scripts/Game/TalkManager.cs

```diff
@@ -1329,51 +1355,72 @@ public string GetNewsOrRumors()
             const int outOfNewsRecordIndex = 1457;
             if (npcData.numAnswersGivenTellMeAboutOrRumors < maxNumAnswersNpcGivesTellMeAboutOrRumors || npcData.isSpyMaster || consoleCommandFlag_npcsKnowEverything)
             {
-                string news = TextManager.Instance.GetLocalizedText("resolvingError");
```

Comment:
This code was simply moved to the "else" branch below. I had to do it this way because of C# variable shadowing rules...